packages/yoastseo/spec/fullTextTests/testTexts/index.js

```diff
@@ -1,176 +1,103 @@
-import portuguesePaper1 from "./pt/portuguesePaper1";
-import portuguesePaper2 from "./pt/portuguesePaper2";
-
-// Czech papers
-import czechPaper1 from "./cs/czechPaper1";
-import czechPaper2 from "./cs/czechPaper2";
-import czechPaper3 from "./cs/czechPaper3";
-
-// Norwegian papers
-import norwegianPaper1 from "./nb/norwegianPaper1";
-import norwegianPaper2 from "./nb/norwegianPaper2";
-import norwegianPaper3 from "./nb/norwegianPaper3";
-
-// Slovak papers
-import slovakPaper1 from "./sk/slovakPaper1";
-import slovakPaper2 from "./sk/slovakPaper2";
-import slovakPaper3 from "./sk/slovakPaper3";
-
-// Greek paper
+// DE: German papers
+import germanPaper from "./de/germanPaper";
+
+// EL: Greek paper
 import greekPaper from "./el/greekPaper";
 
-// Japanese paper
+// EN: English papers
+import englishPaper from "./en/englishPaper";
+
+// ES: Spanish papers
+import spanishPaper from "./es/spanishPaper";
+
+// FA: Farsi papers
+import farsiPaper from "./fa/farsiPaper";
+
+// FR: French papers
+import frenchPaper from "./fr/frenchPaper";
+
+// HE: Hebrew papers
+import hebrewPaper from "./he/hebrewPaper";
+
+// HU: Hungarian papers
+import hungarianPaper from "./hu/hungarianPaper";
+
+// ID: Indonesian papers
+import indonesianPaper from "./id/indonesianPaper";
+
+// IT: Italian papers
+import italianPaper from "./it/italianPaper";
+
+// JA: Japanese paper
 import japanesePaper from "./ja/japanesePaper";
 
+// NB: Norwegian papers
+import norwegianPaper from "./nb/norwegianPaper";
+
+// NL: Dutch papers
+import dutchPaper from "./nl/dutchPaper";
+
+// PL: Polish papers
+import polishPaper from "./pl/polishPaper";
+
+// PT: Portuguese papers
+import portuguesePaper from "./pt/portuguesePaper";
+
+// RU: Russian papers
+import russianPaper from "./ru/russianPaper";
+
+// SK: Slovak papers
+import slovakPaper from "./sk/slovakPaper";
+
+// SV: Swedish papers
+import swedishPaper from "./sv/swedishPaper";
+
+// TR: Turkish papers
+import turkishPaper from "./tr/turkishPaper";
+
+// Papers for comparing performance of stemmers in different languages in the content analysis app.
+import englishPaperForPerformanceTest from "./en/englishPaperForPerformanceTest";
+import spanishPaperForPerformanceTest from "./es/spanishPaperForPerformanceTest";
+import polishPaperForPerformanceTest from "./pl/polishPaperForPerformanceTest";
+
 /**
  * FrenchPaper1 & portuguesePaper3 are temporarily disabled until we figure out why there are small differences
  * in passive voice detection since upgrading from node v10 to the lts version.
  */
 export default [
-	englishPaper1,
-	englishPaper2,
-	englishPaper3,
-	englishPaper4,
-	swedishPaper1,
-	swedishPaper2,
-	germanPaper1,
-	germanPaper2,
-	germanPaper3,
-	frenchPaper2,
-	frenchPaper3,
-	italianPaper1,
-	italianPaper2,
-	italianPaper3,
-	spanishPaper1,
-	spanishPaper2,
-	spanishPaper3,
-	polishPaper1,
-	polishPaper2,
-	polishPaper3,
-	russianPaper1,
-	russianPaper2,
-	russianPaper3,
-	arabicPaper1,
+	englishPaper,
```

Comment:
Interestingly enough, the English paper needs to be the first one exported, because it's used in other files which call `testTexts[0]`.